src/store/sync/message.ts

```diff
@@ -28,6 +41,11 @@ function playSound(): void {
 
 const triggerNotification = (m: SoapIncompleteMessage): void => {
 	const { t } = getBridgedFunctions();
+	const { props, prefs } = getUserSettings();
+	const isShowNotificationEnabled = prefs?.zimbraPrefMailToasterEnabled ?? 'TRUE';
+	const isAudioEnabled = find(props, ['name', 'mailNotificationSound'])?._content ?? 'TRUE';
+	const showAllNotifications = prefs?.zimbraPrefShowAllNewMailNotifications ?? 'FALSE';
```

Comment:
@nubsthead is there any input to change this settings ?

Reply:
zimbraPrefShowAllNewMailNotifications? yes it is in the settings panel